src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java

```diff
@@ -64,9 +70,16 @@ public int runDevAppServerCommand(List<String> args) throws ProcessRunnerExcepti
     command.add(getDevAppServerPath().toString());
     command.addAll(args);
 
+    sysOutCommand(command);
+
     return processRunner.run(command.toArray(new String[command.size()]));
   }
 
+  private void sysOutCommand(List<String> command) {
```

Comment:
This is just nitpicky, but this functions utility doesn't seem as obvious as before, feel like inline is easier to read?